acceptance/run.sh

```diff
@@ -0,0 +1,5 @@
+#!/bin/bash
+
+cd $(dirname $0)/..
+build/builder.sh make install
```

Comment:
or probably elsewhere actually, since they're created by the docker user we may not have permissions.
alternatively: use a tmp dir for certs.